src/libpgmoneta/message.c

```diff
@@ -2655,6 +2749,28 @@ pgmoneta_receive_manifest_file(SSL* ssl, int socket, struct stream_buffer* buffe
       }
       if (msg->kind == 'd' && msg->length > 0)
       {
+
+         if (bucket->burst)
+         {
+            if (msg->length > bucket->burst)
+            {
+               pgmoneta_log_error("the file size(%ld) exceeds the burst capacity of the bucket(%ld).", msg->length, bucket->burst);
+               goto error;
+            }
+
+            while (1)
```

Comment:
I think user should have the option not to use rate limiter, if they don't configure it, or set it to `off`.

Reply:
The default value is 0, and it will not be used if a specific value is not provided.